Description: Do not hardcode prefix in libcomps.pc.in.
Author: Mihai Moldovan <ionic@ionic.de>

This patch SHOULD be upstreamed ASAP!

--- a/libcomps.pc.in
+++ b/libcomps.pc.in
@@ -1,4 +1,4 @@
-prefix=/usr/local
+prefix=@CMAKE_INSTALL_PREFIX@
 includedir=${prefix}/include
 libdir=${prefix}/lib@LIB_SUFFIX@
 
